### PR TITLE
Changed Yggdrasil to work as a base for both SRE and DT team

### PR DESCRIPTION
**Pull request by jaxels10**

### CODE REVIEW

File: yggdrasil/applications/example-guestbook/example-guestbook.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+#domain: hej
```
Comment (jacobcrawford) [inline yggdrasil/applications/example-guestbook/example-guestbook.yaml:1]: Is this file used?

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Looks very good!
Only a few questions :)

File: README.md [hunk 2]
```diff
@@ -15,40 +15,101 @@ The environment repository contains all deployment manifests of the currently de
...
+otherValue: <myValue>
+project: 
+  indentedValue: <indentedValue>
+```
+
 This PR will then need to be approved by the cluster development team, before it is merged into Yggdrasil. When this is merged, ArgoCD will automatically deploy the application onto the cluster. 
 When the deployment is done, ArgoCD will poll the environment repository every 3 minutes, to check for changes to the application. 
 
+## Installing chart
+To install Yggdrasil, you first need to navigate to the nidhogg directory and run a helm dependency update.
```
[the remark this answers is not in the document]
Comment (jaxels10) [reply README.md:110]: Haha sorry, that is the command xD "helm dependency update"

[the remark this answers is not in the document]
Comment (jaxels10) [reply README.md]: Changed it now to be a command

File: yggdrasil/templates/namespace.yaml [hunk 3]
```diff
@@ -1,10 +1,11 @@
-
 {{- $root := . -}}
-{{- range $path, $_ := .Files.Glob "services/**/config.yaml" }}
-    {{ $project := $root.Files.Get $path | fromYaml }}
+{{- range $path, $_ := .Files.Glob "**/**/config.yaml" }}
+  {{ $project := $root.Files.Get $path | fromYaml }}
```
[the remark this answers is not in the document]
Comment (jaxels10) [reply yggdrasil/templates/namespace.yaml:3]: Hmm, it is a variable that stores a single object containing only 1 project, which is why it is not plural

File: yggdrasil/applications/example-guestbook/example-guestbook.yaml (hunk 1 above)
Comment (jaxels10) [reply yggdrasil/applications/example-guestbook/example-guestbook.yaml:1]: Only for testing purposes, the example-guestbook will be deleted soon

**Review by jacobcrawford — APPROVED** [review]
looks good.

**Pull request merged**